reana_commons/serial.py

```diff
@@ -81,7 +81,8 @@
 }
 
 
-def serial_load(workflow_file, specification, parameters=None, original=None):
+def serial_load(workflow_file, specification, parameters=None, original=None,
```

Comment:
Who is using the `**kwargs`?

Reply:
Same as explained here https://github.com/reanahub/reana-client/pull/400#discussion_r403120522